src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -456,3 +452,14 @@ def validate_namespace(namespace):
         return True
     else:
         return False
+
+
+
+def load_db_config():
```

Comment:
Use 1 or 2 empty lines between functions.

Reply:
This function is moved to sonic_utilities in the PR https://github.com/Azure/sonic-utilities/pull/1712